core/src/components/toast/toast.tsx

```diff
@@ -28,9 +28,12 @@ export class Toast implements ComponentInterface, OverlayInterface {
   animation: Animation | undefined;
 
   @Prop({ connect: 'ion-animation-controller' }) animationCtrl!: HTMLIonAnimationControllerElement;
+
   @Prop({ context: 'config' }) config!: Config;
 
-  /** @internal */
+  /**
```

Comment:
I was actually moving to:
```
/** @internal */
```
thoughts?

Reply:
I'd just prefer if it were in the same format as the other comments, idk